src/app/tests/AppTestContext.cpp

```diff
@@ -57,6 +58,7 @@ void AppContext::Shutdown()
     Access::ResetAccessControlToDefault();
 
     chip::app::InteractionModelEngine::GetInstance()->Shutdown();
+    chip::DeviceLayer::PlatformMgr().Shutdown();
```

Comment:
Actually, we may also have existing code that inits the platform manager outside of this class.  Needs to be checked.

Reply:
The nrfconnect failure was caused by repeated CHIP stack init/shutdown sequence in one test application (out/nrf-native-posix-64-tests/zephyr/zephyr.exe), however, in case of mbedTLS crypto backend the shutdown process does not release all resources... The problem is with `mbedtls_entropy_add_source()` which does not have `*_remove_source()` counterpart. As a consequence, this function returns `MBEDTLS_ERR_ENTROPY_MAX_SOURCES` after few init/shutdown iterations. I've added a commit which mitigates that, however, it accesses "private" struct members...